src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml

```diff
@@ -100,13 +100,11 @@
 			<StaticResource x:Key="ComboBoxPlaceHolderForegroundFocused" ResourceKey="OnSurfaceLowBrush" />
```

Comment:
@Arieldelossantos please the XAML changes separated in another PR so we can backport them properly without the C# changes (cc @kazo0, @Xiaoy312).